schema-loader/README.md

```diff
@@ -294,3 +294,61 @@ However, the following types are converted differently when they are used as a p
 | BLOB | VARBINARY(64) | | RAW(64) | |
 
 If this data type mapping doesn't match your application, please alter the tables to change the data types after creating them with this tool.
+
+
+## Using Schema Loader in your program
+You can check `schema-loader` from [maven central repository](https://mvnrepository.com/artifact/com.scalar-labs/scalardb-schema-loader).
+For example in Gradle, you can add the following dependency to your build.gradle. Please replace the `<version>` with the version you want to use.
+```
+dependencies {
+    implementation group: 'com.scalar-labs', name: 'scalardb-schema-loader', version: '<version>'
+}
+```
+
+### Create/Delete tables:
+The main component of `schema-loader` is `SchemaOperator`, we will use an instance of this class for creating/deleting schema.
+There are several ways to create `SchemaOperator` instance using [`SchemaOperatorFactory`](https://github.com/scalar-labs/scalardb/blob/master/schema-loader/src/main/java/com/scalar/db/schemaloader/core/SchemaOperatorFactory.java)
+
+The typical way is passing a path to the Scalar DB configuration file `database.properties`
+
+```java
+public class SchemaLoaderSample {
+  public static int main(String... args) throws IOException, SchemaOperatorException {
+    String configPath = "database.properties";
+    String schemaFile = "sample_schema.json";
+
+    Map<String, String> metaOptions = new HashMap<>();
+
+    metaOptions.put(
+        CassandraAdmin.REPLICATION_STRATEGY, ReplicationStrategy.SIMPLE_STRATEGY.toString());
+    metaOptions.put(CassandraAdmin.COMPACTION_STRATEGY, CompactionStrategy.LCS.toString());
+    metaOptions.put(CassandraAdmin.REPLICATION_FACTOR, "1");
+
+    metaOptions.put(DynamoAdmin.REQUEST_UNIT, "1");
+    metaOptions.put(DynamoAdmin.NO_SCALING, "true");
+    metaOptions.put(DynamoAdmin.NO_BACKUP, "true");
+
+    SchemaOperator operator = SchemaOperatorFactory.getSchemaOperator(configPath, true);
```

Comment:
Can you please change this interface to something like the following:
```
SchemaLoader.load(File jsonFile, Map<String, String> options);
SchemaLoader.load(String jsonString, Map<String, String> options);
```

Reply:
Thanks for the suggestion. Applied it in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/d1cd8db9cb47fbec4edf0cfce6daf996a711ba5f)